search-api/src/main/scala/no/ndla/searchapi/service/search/IndexService.scala

```diff
@@ -235,7 +233,7 @@ trait IndexService[D <: Content](using
 
       if (indexRequests.nonEmpty) {
         val response = e4sClient.execute {
-          bulk(indexRequests)
+          bulk(indexRequests.filter(_.isDefined).flatten)
         }
```

Comment:
Burde vel få med den filtreringen i den if-checken også?


```suggestion
      val filteredRequests = indexRequests.flatten
      if (filteredRequests.nonEmpty) {
        val response = e4sClient.execute {
          bulk(filteredRequests)
        }
```